heartbeat/heartbeat.yml

```
@@ -128,8 +128,15 @@ output.elasticsearch:
 
 #================================ Processors =====================================
 processors:
-  - add_observer_metadata: ~
+  - add_observer_metadata: 
+    # Optional, but recommended geo settings for the location heartbeat is running in 
+    # geo: 
```

```suggestion
    #geo: 
```
Sorry, still not happy with the indentation. I'm mainly worried that users uncommenting entries will mess things up.